client/src/components/assessments/AssessmentResultsTable.js

```diff
@@ -0,0 +1,325 @@
+import AggregationWidget from "components/AggregationWidgets"
+import AddAssessmentModal from "components/assessments/AddAssessmentModal"
+import { ReadonlyCustomFields } from "components/CustomFields"
+import Fieldset from "components/Fieldset"
+import { Formik } from "formik"
+import LinkTo from "components/LinkTo"
+import _isEmpty from "lodash/isEmpty"
+import PropTypes from "prop-types"
+import React, { useState } from "react"
+import { Button, Table } from "react-bootstrap"
+import "components/assessments/AssessmentResultsTable.css"
+
+/* The AssessmentResultsTable component displays the results of two types of
+ * assessments made on a given entity and subentities:
+ * - aggregation of the measurements made on the entity/subentities when
+ *   working on them in relation to another type of entity (example:
+ *   assessments made on tasks, while filling  report related to the tasks);
+ *   the definition of these assessments is to be found in
+ *   entity.customFields.assessmentDefinition
+ * - display of the last monthly assessment made on the entity/subentities
+ *   as a conclusion about the given period of time;
+ *   the definition of these assessments is to be found in
+ *   entity.periodAssessmentConfig()
+ */
+
+const AssessmentsTableHeader = ({ periods }) => (
+  <thead>
+    <tr key="periods">
+      <>
+        {periods.map(period => (
+          <th key={period.start}>{period.start.format("MMM-YYYY")}</th>
```

Comment:
Extract `"MMM-YYYY" as a constant or perhaps even move to the `anet.yml` dictionary?

Reply:
Let's add this as debt. We have a requirement to have assessment periods of different time windows (weekly/monthly/bi-weekly/yearly... etc). This could be addressed in such a PR

Reply:
Well, at least a `const PERIOD_FORMAT = "MMM-YYYY"` at the top would be nicer than having this fixed string in three places.